plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -580,4 +581,17 @@ public IcebergConfig setBucketExecutionEnabled(boolean bucketExecutionEnabled)
         this.bucketExecutionEnabled = bucketExecutionEnabled;
         return this;
     }
+
+    public boolean isFileBasedConflictDetectionEnabled()
+    {
+        return fileBasedConflictDetectionEnabled;
+    }
+
+    @Config("iceberg.file-based-conflict-detection")
```

Comment:
`@ConfigHidden`  - this is a kill-switch as i see it .

Reply:
That doesn't mean it has to be hidden, it should just be removed eventually